db/schema.rb

```diff
@@ -416,6 +416,14 @@
   end
   add_index "course_material_folders", ["parent_id", "name"], name: "index_course_material_folders_on_parent_id_and_name", unique: true, case_sensitive: false
 
+  create_table "course_material_folder_downloads", force: :cascade do |t|
+    t.integer  "folder_id",  null: false, index: {name: "fk__course_material_folder_downloads_folder_id"}, foreign_key: {references: "course_material_folders", name: "fk_course_material_folder_downloads_folder_id", on_update: :no_action, on_delete: :no_action}
+    t.integer  "status",     default: 0, null: false
+    t.json     "result"
```

Comment:
@lowjoel This structure is similar to answer_auto_grading, the result here will store info like the url of zip file, current progress of downloading, etc